app/config/findbugs.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+
+<!--
+FindBugs exclude filter
+Severity: HARD
+-->
+<FindBugsFilter>
```

Comment:
>the build hasn't failed. Meaning that find bugs is not parsing those files at all

this is used to exclude these files from checking syntax bugs right? @iamareebjamal